README.md

```diff
@@ -10,7 +10,20 @@ If you do not have an ARM compiler you will need to install one:
 sudo apt install gcc-arm-none-eabi
 ```
 
-This component needs `CMake` >= 3.19, `colcon` and other Python 3 packages in order to build micro-ROS packages:
+This component needs `CMake` >= 3.19.
+If required, you can install the latest cmake version with the following commands:
+
+```bash
```

Comment:
I'm not sure if we should include this.
Which of those packages are Mbed dependencies? In that case, we should leverage Mbed instructions in order to not provide outdated information.

Regarding CMake, do Mbed provide instructions for updating CMake to the required version? If not, we should link to Kitware official instructions, also in order to not provide outdated instructions.

Reply:
All packages are `mbed` dependencies. I did not find `CMake` instructions on the mbed website. I got these exact instructions from @Acuadros95

Reply:
If they are mbed deps we should link the mbed installation instructions.